Model/SdkProxy.php

```diff
@@ -91,23 +106,61 @@ private function getSubject(): GraphQlSdk
      */
     public function getMethods(string $value = null, string $currency = null, ?array $inputOptions = null): array
     {
-        // If value & currency are both not null, use separate method.
-        if ($value !== null && $currency !== null) {
-            return $this->getMethodsByValueAndCurrency($value, $currency, $inputOptions);
-        }
+        $identifier = $value . '_' . $currency;
+
+        if (!$this->cache->load($identifier)) {
```

Comment:
Very minor nitpick: 

Is it worth it to save `$this->cache->load($identifier)` to a variable so you don't have to load from the cache twice in the code path where its there?

Reply:
Nice one, resolved